### DISCUSSION TITLE
MilvusException: (code=65535, message=invalid character 'N' looking for beginning of value)

### DISCUSSION THREAD

**Discussion by n1tz53** [milvus-io/milvus#36638] [Q&A and General discussion] [answered]
I'm trying to add dynamic fields into the collection using python SDK, I get the exception mentioned as title, can someone help ?

**Comment by yhmo** [comment, marked as answer]
The error indicates the milvus server could not parse JSON format string with some special characters. Double-check the values you input to the dynamic field.
